.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+on: [push]
+
+env:
+  COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+  GRADLE_OPTS: -Xmx3200m -XX:+HeapDumpOnOutOfMemoryError -Dkotlin.incremental=false -Dkotlin.compiler.execution.strategy="in-process" -Dorg.gradle.daemon=false
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: Brightspace/third-party-actions@actions/checkout
+
+    - name: Setup Java
+      uses: Brightspace/third-party-actions@actions/setup-java
+      with:
+        java-version: '8'
+
+    - name: Run Tests
+      run: |
+        ./gradlew assemble
+        ./gradlew check
```

Comment:
These seem like reasonable tasks, but not sure where they came from - was Travis not doing this? Or maybe they're run implicitly with `jacocoTestReport`?

Reply:
Travis was, implicitly, though running `jacocoTestReport` wasn't sufficient in this context.